src/containers/ProjectContentsContainer.jsx

```diff
@@ -24,7 +27,9 @@ class ProjectContentsContainer extends Component {
 
   componentDidMount() {
     const { actions } = this.props;
-    return actions.fetchProjectContents(this.props.params.project_id);
+    const type = this.props.params.resource_type;
+    const id = type ? this.props.params.resource_id : this.props.params.project_id;
+    return actions.fetchResource(id, type);
```

Comment:
Sorry, just some cruft. However, I have experienced problems in triggering actions when `return` weren't used...